netlogo-gui/src/main/app/InterfacePanel.scala

```diff
@@ -331,11 +333,11 @@ class InterfacePanel(val viewWidget: ViewWidgetInterface, workspace: GUIWorkspac
   def keyPressed(evt: KeyEvent): Unit = {
```

Comment:
Why do you need to handle the shortcuts yourself? Why not deleting the whole `keyPressed` functionality and allowing the standard menu-items in the Edit menu to do the undo/redo? you're calling setCurrentManager, so I guess it should work fine